src/js/common/store/index.js

```diff
@@ -11,6 +12,10 @@ export const commonReducer = {
   })
 };
 
+export function renderCommonComponents(commonStore) {
```

Comment:
It feels weird to me for this helper to be in this file, which is focused on Redux store things.

Reply:
Yeah I know what you mean. It felt weird to me to put it anywhere really. Any suggestions? I didn't want to mix it with all of the components.

Reply:
Good question. Somewhere in /common. It could be an `init-components` file in the main common directory?

Reply:
Alternatively, we could change the name of this file and treat it as the common set up code?

Reply:
The second idea sounds good to me. Makes sense because if the common store is going to be used then we'll most likely want the common components along with it